setup.py

```diff
@@ -18,19 +17,13 @@ def readfile(filename, split=False):
 readme = readfile("README.rst", split=True)[3:]  # skip title
 # For requirements not hosted on PyPi place listings
 # into the 'requirements.txt' file.
-requires = []  # minimal requirements listing
+requires = [
+    # minimal requirements listing
+    'opencmiss.utils',
+    'ZincPythonTools',
+]
```

Comment:
We can also use the following instead 
```
['opencmiss.utils @ https://api.github.com/repos/OpenCMISS-Bindings/opencmiss.utils/tarball/master', 
'ZincPythonTools' @  https://api.github.com/repos/OpenCMISS-Bindings/ZincPythonTools/tarball/master']
```
This should work for downstream users such as scaffold-web-demo.